post_installation_configuration/coreos-layering.adoc

```diff
@@ -82,19 +146,45 @@ Because libreswan requires additional RHEL packages, the image must be built on
 ** SSH Key management packages.
 --
 +
-.Example Containerfile to apply a third-party package from EPEL
+.Example on-cluster Containerfile to apply a third-party package from EPEL
 [source,yaml]
 ----
-include::https://raw.githubusercontent.com/openshift/rhcos-image-layering-examples/master/htop/Containerfile[]
+# Get RHCOS base image of target cluster `oc adm release info --image-for rhel-coreos`
+# hadolint ignore=DL3006
+FROM configs AS final
+
+#Enable EPEL (more info at https://docs.fedoraproject.org/en-US/epel/ ) and install htop
+RUN rpm-ostree install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
+    rpm-ostree install htop && \
+    ostree container commit
 ----
 +
-.Example Containerfile to apply a third-party package that has {op-system-base} dependencies
+.Example out-of-cluster Containerfile to apply a third-party package from EPEL
 [source,yaml]
 ----
-include::https://raw.githubusercontent.com/openshift/rhcos-image-layering-examples/master/fish/Containerfile[]
+include::https://raw.githubusercontent.com/openshift/rhcos-image-layering-examples/master/htop/Containerfile[]
 ----
 +
 This Containerfile installs the Linux fish program. Because fish requires additional RHEL packages, the image must be built on an entitled {op-system-base} host.
++
+.Example on-cluster Containerfile to apply a third-party package that has {op-system-base} dependencies
+[source,yaml]
+----
+# Get RHCOS base image of target cluster `oc adm release info --image-for rhel-coreos`
+# hadolint ignore=DL3006
+FROM configs AS final
+
+# RHEL entitled host is needed here to access RHEL packages
```

Comment:
issue: For RHEL entitlements to work, right now, they need to copy the `etc-pki-entitlement` secret into the MCO namespace. We're still doing some due diligence with the RHEL entitlements team right now, so that process might change in the future.

suggestion: Add a one-time setup instruction indicating that they would need to copy the entitlement secret if they want to use RHEL entitlements.